src/Kernel.php

```diff
@@ -45,12 +45,12 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
         $loader->load($confDir . '/{services}_' . $this->environment . self::CONFIG_EXTS, 'glob');
     }
 
-    protected function configureRoutes(RouteCollectionBuilder $routes): void
```

Comment:
You can remove the whole method, it's on the MicroKernelTrait

Reply:
Nah.